src/Rings/mpoly-localizations.jl

```diff
@@ -3185,3 +3185,24 @@ end
 
 dim(R::MPolyLocRing{<:Field, <:FieldElem, <:MPolyRing, <:MPolyElem, <:MPolyComplementOfPrimeIdeal}) = nvars(base_ring(R)) - dim(prime_ideal(inverted_set(R)))
 
+########################################################################
+# Localizations of graded rings                                        #
+########################################################################
```

Comment:
I was happily surprised that localizing graded rings does not seem to pose any problems so far. The code is sufficiently generic so that it just works. But I am wondering whether there are some more functions which we should provide for such localizations, see below.